numba_dppy/compiler.py

```diff
@@ -31,12 +31,11 @@
 
 from . import spirv_generator
 
-import os
 from numba.core.compiler import DefaultPassBuilder, CompilerBase
 from numba_dppy.dppy_parfor_diagnostics import ExtendedParforDiagnostics
+from numba_dppy.config import DEBUG
```

Comment:
DEBUG variable should be removed from numba_dppy.compiler.
It will require replace callers code.
It is for separate PR.